camunda/v2/examples/task-deletion/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"os/signal"
+
+	"github.com/blacklane/go-libs/camunda/v2"
+	"github.com/blacklane/go-libs/logger"
+)
+
+const (
+	url        = "http://localhost:8080"
+	processKey = "example-process"
+	businessKey = "US_ee14f205-dccc-4d6f-a3ae-5d20a0015d3b"
+)
+
+func main() {
+	// catch the signals as soon as possible
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt) // a.k.a ctrl+C
+
+	log := logger.New(
+		os.Stdout,
+		"camunda-sample-client",
+		logger.WithLevel("debug"))
+
+	credentials := camunda.BasicAuthCredentials{
+		User:     "Bernd",
+		Password: "password",
+	}
+
+	// camunda stuff
+	client := camunda.NewClient(url, processKey, http.Client{}, credentials)
+
+	err := client.DeleteTask(context.Background(), businessKey)
+	if err != nil {
+		log.Err(err).Msg("Failed to Delete Task")
+	}
```

Comment:
here we can join the execution with the error validation and log a `fatal` to stop 
running the app and prevent `log.Info().Msg("successful!")` from being displayed
```suggestion
	if err := client.DeleteTask(context.Background(), businessKey); err != nil {
		log.Fatal().Err(err).Msg("Failed to Delete Task")
	}
```